node-zerox/src/utils.ts

```diff
@@ -94,7 +95,9 @@ export const downloadFile = async ({
 }): Promise<{ extension: string; localPath: string }> => {
   // Shorten the file name by removing URL parameters
   const baseFileName = path.basename(filePath.split("?")[0]);
```

Comment:
If not using the `baseFileName` for the localPath, we can probably get the extension name from just the filePath itself and remove this line.
```
const fileNameExt = path.extname(filePath);
const localPath = path.join(tempDir, uuidv4() + fileNameExt);
```